src/skin/legacy/tooltips.cpp

```diff
@@ -1005,6 +1005,12 @@ void Tooltips::addStandardTooltips() {
     add("show_intro_outro_cues")
             << tr("Show/hide intro & outro markers and associated buttons.");
 
+    add("keep_consistent_waveform_breadth") << tr(
+            "Ensure all waveforms to have the same height across all channels. "
+            "By default, when displaying the stem controls, waveform for "
+            "channel that have no stem may render with a shorter breadth in "
+            "order to honor the waveform container size you have requested..");
```

Comment:
I'm sure the current tooltip is confusing :D 

Please suggest a better wording, if you can think of one!